src/error.rs

```diff
@@ -1036,6 +1038,7 @@ impl ToDiagnostic<FileId> for EvalError {
                     .with_labels(labels)
                     .with_notes(vec![msg.clone()])]
             }
+            EvalError::ParseError(parse_error) => parse_error.to_diagnostic(files, contract_id),
```

Comment:
I think this is fine for now, but in the future we may want to display an error message that makes it explicit that the error was somehow "dynamic", i.e. happened during evaluation. The reason is that otherwise, it can be confusing to toggle a parse error on and off depending on what get evaluated at runtime (I imagine user expect parse error to always happen).